db/seeds/tutorials.yml

```diff
@@ -170,7 +170,7 @@
         <strong>Pro-Tips:</strong>
         <ul>
           <li>A POV is your Personally Owned Vehicle, and in this context, a Mobile Home is an RV.</li>
-          <li>Not all locations, in particular OCONUS, allow you to ship a motorcycle and a POV. Check out the Consignment Instruction Guide for motorcycle shipping information on each location.</li>
+          <li>Not all locations, in particular OCONUS, allow you to ship a motorcycle and a POV. Check out the <a href="https://tops.ppcigweb.sddc.army.mil/ppcig/menu/query/country.do">Consignment Instruction Guide</a> for motorcycle shipping information on each location.</li>
```

Comment:
The link to the Consignment Instruction Guide currently errors with "Bad Gateway". 

Also, according to _USTRANSCOM Personal Property Advisory # 18-0028_ which I received in my e-mail today:

> The SDDC PPCIG was shut down as of 30 November 2017.

Reply:
@LillyMadigan and @jde1984 are trying to obtain a copy of this so we can host it, or point to something else. I can remove this sentence for now.